walkthroughs/week-4-libraries/maps/examples/hello-world/src/main/webapp/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <title>Hello Google Maps</title>
-  <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY"></script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBTLLpFbDtVnUOs0KSEiqBUpq5Dod85Nps"></script>
```

Comment:
Do you need to change this? You should only need this in your own app right?

Reply:
we needed to change it in order to run it on the dev server so we could see what it looks like, should I change it back to "YOUR_API_KEY"?

Reply:
What I mean is why do you run the maps/examples/hello-world/src/main/webapp/index.html? Shouldn't you only run portfolio/src/main/webapp/index.html?

Reply:
I run them on the dev server so I can see what the example sites look like and how they work

Reply:
Right, that is fine but it seems unnecessary to check-in these changes, since now your own servers worked. (Change it back to "YOUR_API_KEY" sounds reasonable)